### ISSUE TITLE
Lost in the workflow for coding and editing codes 

### ISSUE THREAD

**Issue by SamHames** [cproctor/qualitative-coding#65]
I'm really enjoying reviewing this package, but I'm getting stuck understanding what the day to day workflow is to use it. I can see all the pieces clearly outlined in the documentation, but the vignettes don't go into enough detail to make me feel confident enough to put it together and apply it to something new.

I'd greatly appreciate a more detailed worked example in line with the more detailed examples for exploring an existing project. Even something like showing the commands to start from scratch, importing a two paragraph document, coding it, then working through the functionality for code grouping would be really useful for contextualising the commands and how to put them together in the context of a research project.

**Comment by cproctor** [comment]
Thanks for the feedback. 

I have just substantially reworked the [Create a new project](https://qualitative-coding.readthedocs.io/en/latest/manuscript.html#create-a-new-project) section; I hope it is easier to follow. Following your suggestion, I elaborated this section in a similar way to [Explore an existing project](https://qualitative-coding.readthedocs.io/en/latest/manuscript.html#explore-an-existing-project). I also streamlined this section by moving some discussion of other possibilities into a new section, [Advanced patterns](https://qualitative-coding.readthedocs.io/en/latest/manuscript.html#advanced-patterns).
